code/controllers/subsystem/processing/networks.dm

```diff
@@ -1,51 +1,340 @@
-PROCESSING_SUBSYSTEM_DEF(networks)
+#define DEBUG_NETWORKS
+
+
+SUBSYSTEM_DEF(networks)
```

Comment:
Virtually every proc you added in this subsystem does not meet our [code documentation standards](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md), please add or improve the documentation here.